examples/java/dynatracesdk-draft/src/main/java/com/tracing/TraceExample.java

```diff
@@ -0,0 +1,202 @@
+		}
+	
+		@Override
+		public void error(String message) {
+			System.out.println(message);
+		}
+	}
+
+	private static void logSdkStatus() {
+		switch (oneAgentSdk.getCurrentState()) {
+		case ACTIVE:
+			System.out.println("dynatrace sdk ACTIVE");
+			break;
+		case PERMANENTLY_INACTIVE:
+			System.out.println("dynatrace sdk PERMANENTLY_INACTIVE");
+			break;
+		case TEMPORARILY_INACTIVE:
+			System.out.println("dynatrace sdk TEMPORARILY_INACTIVE");
+			break;
+		default:
+			System.out.println("dynatrace sdk status unknown");
+			break;
+		}
+	}
+
+	private static void initClient() {
+		Thread thread = new Thread(new Runnable() {
+
+			@Override
+			public void run() {
+				int pos = 0;
+				String[] pathList = { "pathA", "pathB" };
+				for (;;) {
+					try {
+						// call the server in a loop
+						System.out.println("Client is calling");
+						pos = (pos + 1) % 2;
+						URL url = new URL("http://localhost:8000/" + pathList[pos]);
+						HttpURLConnection con = (HttpURLConnection) url.openConnection();
+						con.setRequestMethod("GET");
+						BufferedReader in = new BufferedReader(new InputStreamReader(con.getInputStream()));
+						String inputLine;
+						StringBuffer content = new StringBuffer();
+						while ((inputLine = in.readLine()) != null) {
+							content.append(inputLine);
+						}
+						in.close();
+					} catch (Exception e) {
+						System.err.println("Failed to talk to server");
+						System.err.println(e.toString());
+					}
+					try {
+						Thread.sleep(1000);
+					} catch (InterruptedException ex) {
+						break;
+					}
+				}
+			}
+		});
+		thread.start();
+	}
+
+	private static void initServer() throws Exception {
+		HttpServer server = HttpServer.create(new InetSocketAddress(8000), 0);
+		server.createContext("/pathA", new PathAHandler());
+		server.createContext("/pathB", new PathBHandler());
+		server.setExecutor(null); // creates a default executor
+		server.start();
+	}
+
+	// server side handlers
+
+	static class PathAHandler implements HttpHandler {
+		WebApplicationInfo webApplicationInfo = oneAgentSdk.createWebApplicationInfo("PathAHandler", "", "/");
+
+		@Override
+		public void handle(HttpExchange t) throws IOException {
+			// traces an incoming request with dynatrace sdk
+			// trace-context is read automatically
+			IncomingWebRequestTracer tracer = oneAgentSdk.traceIncomingWebRequest(webApplicationInfo, t.getRequestURI().toString(), "GET");
+
+			// record header properties of request (optional)
+			for (String key : t.getRequestHeaders().keySet()) {
+				for(String value : t.getRequestHeaders().get(key)) {
+					tracer.addRequestHeader(key, value);
+				}
+			}
+
+			tracer.start();
+			try {
+				String response = "This is path A";
+				t.sendResponseHeaders(200, response.length());
+				OutputStream os = t.getResponseBody();
+				os.write(response.getBytes());
+				System.out.println("Path A was called.");
+				os.close();
+			} catch (Throwable e) {
+				tracer.error(e.getMessage());
```

Comment:
in java API we provide tracer.error(String) and tracer.error(Throwable). i suggest to use the second one in catch() blocks for java. (as it will add exception attachment with full stackstrace no the node)